examples/matlab_mex/pendulum_on_cart_model/example_closed_loop.m

```diff
@@ -45,56 +43,48 @@
 end
 
 
-
-%% arguments
-compile_mex = 'true';
-codgen_model = 'true';
+%% options
+compile_mex = 'true'; % true, false
+codgen_model = 'true'; % true, false
 % simulation
-sim_method = 'irk';
-sim_sens_forw = 'false';
+sim_method = 'irk'; % erk, irk, irk_gnsf
+sim_sens_forw = 'false'; % true, false
 sim_num_stages = 4;
 sim_num_steps = 4;
 % ocp
 param_scheme = 'multiple_shooting_unif_grid';
 ocp_N = 100;
-nlp_solver = 'sqp';
-%nlp_solver = 'sqp_rti';
-%nlp_solver_exact_hessian = 'false';
+nlp_solver = 'sqp'; % sqp, sqp_rti
 nlp_solver_exact_hessian = 'true';
-%regularize_method = 'no_regularize';
-%regularize_method = 'project';
-regularize_method = 'project_reduc_hess';
-%regularize_method = 'mirror';
-%regularize_method = 'convexify';
+regularize_method = 'project_reduc_hess'; % no_regularize, project,...
+    % project_reduc_hess, mirror, convexify
 nlp_solver_max_iter = 100;
 qp_solver = 'partial_condensing_hpipm';
-%qp_solver = 'full_condensing_hpipm';
+        % full_condensing_hpipm, partial_condensing_hpipm
 qp_solver_cond_N = 5;
-qp_solver_cond_ric_alg = 0;
-qp_solver_ric_alg = 0;
 qp_solver_warm_start = 0;
-ocp_sim_method = 'erk';
-%ocp_sim_method = 'irk';
+qp_solver_cond_ric_alg = 0; % HPIPM specific? what does it stand for?
```

Comment:
this chooses if the hessian is factorized or not in the condensing

Reply:
Interesting, but I dont have a clear picture of how it works.
I would add a comment:
`HPIPM specific: 0: dont factorize hessian in the condensing; 1: factorize`
okay?

Reply:
this is not HPIPM specific: all solvers use some sort of condensing

Reply:
If the Hessian is positive definite, then factorize the Hessian is usually faster. If the Hessian in not positive definite, then the factorization can not be used.